packages/notifi-react-card/lib/hooks/useNotifiSubscribe.ts

```diff
@@ -204,11 +206,19 @@ export const useNotifiSubscribe: ({
         .catch((_e) => {
           /* Intentionally empty */
         });
+    } else if (demoPreview) {
+      // Mockup info for demo preview card
+      setEmail(defaultDemoConfigV1.name);
+      setPhoneNumber('+101234567890');
+      setTelegramId(defaultDemoConfigV1.id!);
```

Comment:
let's just have an early return if `demoPreview` is not null, and rely on the initial values passed to the demoPreview